docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+
+<div class="example-map">
+    <iframe
+        id="animation-step-7"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-7.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+### Create a basic animation
+
+Before we begin, it is important to understand the general syntax for animating data in CARTO VL:
+
+`filter: animation(input, duration, fade(fadeIn, fadeOut))`
+
+
+We will walk through each part in more detail throughout this guide.
+
+Using the syntax above, let’s create a basic animation where points appear and disappear on a map. 
+
+We will use the attribute `date_time` (a record of time when a data point was collected) as our input, set the duration of the animation to `10` seconds and set both fade parameters to `1`. 
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 10, fade(1,1))
+`);
+```
+
+The result is an animation that cycles through all of the records in the data over 10 seconds. Each point will take 1 second to become visible (fade in) and another second to disappear (fade out).
+
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 10, fade(1,1))
+`);
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-0"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-0.html"
+        width="100%"
+        height="500"a
+        frameBorder="0">
+    </iframe>
+</div>
+
+#### The filter property
+
+
+Now that we have a basic map of animated points, let’s get a deeper understanding of the different pieces that make up the animation expression to fine-tune the visualization.
+
+In the example above, the animation expression is set to the `filter` property (for more information see the[vizSpec](https://carto.com/developers/carto-vl/reference/#vizspec)). The convention in CARTO VL is that `0` represents the boolean value `false`, (the absence of or an *off* state), while `1` represents `true` (the presence of or an *on* state). In the context of an  animation expression assigned to a `filter`, any feature that has a  value of `0` will be “filtered-out” and not shown while any feature that meets the filter, is assigned a value of `1` and is shown. 
+
+You can think of the  animation expression like a clock, cycling through each record of the input.  Any time there is a match, ,  the feature is drawn.  Now that we’ve gone through a more in-depth discussion of how features draw, let’s take a look at the next set of animation parameters are used to define the property, speed, and transition between animated features.
+
+#### Input
```

Comment:
Maybe it's better to just use bulleted points for these 3 items, or even a table. I find it would be a clearer way of displaying them